src/pool/srv_target.c

```diff
@@ -244,7 +244,8 @@ flush_ult(void *arg)
 		} else if (sched_req_space_check(child->spc_flush_req) == SCHED_SPACE_PRESS_NONE) {
 			sleep_ms = 500;
 		} else {
-			sleep_ms = (nr_flushed < nr_flush) ? 50 : 0;
+			/* Never voluntarily yield the CPU when experiencing space pressure! */
```

Comment:
This means once there are space pressure and even we could not reclaim anything. flush ult will be sched frequently. if this hurts performance(maybe not). vos_pool_flush() might return some hints(eg there are more entries going to be reclaimed), or just change it to:

sleep_ms = nr_flushed > 0 ? 0 : 50